generators/python_functions/html_unescape/config.py

```diff
@@ -0,0 +1,15 @@
+from util.configs import build_generator_function_config
+from util.enums import State
+from . import html_unescape, INPUT_EXAMPLE
+
+
+def get_config():
+    return build_generator_function_config(
+        function=html_unescape,
+        input_example=INPUT_EXAMPLE, 
+        data_type="text",
+        issue_id=233,
+        tabler_icon="IconHtml",
+        min_refinery_version="1.9.0",
```

Comment:
The min refinery version is based on when needed requirements/ libraries are added. Is the HTML library already included in Python? If so, you can change this to 1.7.0. 👍

Reply:
How can I figure this out in the future? `_template` says `min_refinery_version="x.x.x", # you need to look this up in the issue`, but I'm not sure where to look.

Reply:
If you add a new library to the requirements.txt of bricks, it's the next version of refinery, because the dev team needs to add the library to the env in refinery. If you don't add a new library, you can just write 1.7.0. Or you can just leave this to the Kern AI team.